mitosheet/mitosheet/tests/test_parse.py

```diff
@@ -76,6 +76,24 @@ def get_string_data_for_df(columns: List[Any], length: int) -> Dict[Any, Any]:
         set([]),
         set([])
     ),
+    (
+        '=TRUE',
+        'B',
+        0,
+        pd.DataFrame(get_string_data_for_df(['B'], 2)),
+        'df[\'B\'] = True',
+        set([]),
+        set([])
+    ),
+    (
+        '=true',
+        'B',
+        0,
+        pd.DataFrame(get_string_data_for_df(['B'], 2)),
+        'df[\'B\'] = True',
+        set([]),
+        set([])
+    ),
```

Comment:
One additional test to see how it performs if TRUE is a column header, then LGTM!

Reply:
Also the column header true. I'm fine if we detect this as the column header or as the constant -- let's just be consistent.